dask/kubernetes/specs/worker-spec.yaml

```diff
@@ -1,20 +1,28 @@
+apiVersion: v1
 kind: Pod
 metadata:
   labels:
+    cluster_type: dask
     dask_type: GPU_worker
 spec:
   restartPolicy: Never
   containers:
-    - image: gcr.io/[YOUR_GCP_PROJECT]/[YOUR_USER_ID]/dask-unified:21.06
-      imagePullPolicy: IfNotPresent
+  # TODO: update image to official image
+    - image: isvoid/rapidsai-cloud-worker:22.06-cuda11.5-runtime-ubuntu20.04-py3.9
+      imagePullPolicy: Always
+      env:
+      - name: DISABLE_JUPYTER
+        value: "true"
       args: [ dask-cuda-worker, $(DASK_SCHEDULER_ADDRESS), --rmm-managed-memory ]
       name: dask-cuda-worker
+      # Optimized against GCP n1-standard-4 P100 nodes to increase occupancy.
```

Comment:
Your cluster create uses A100s but this says P100s. Do you mean you tested against P100s when writing this?

Reply:
Yeah, unfortunately the optimization didn't go all the way to the bottom. The A100 nodes also looks dramatically different from a P100 node. In this PR I should just focus on getting the notebook runnable again. I think there maybe several follow ups to this PR including refactoring the notebook contents and exploring the optimal resource allocations.

Reply:
Will just remove this line.